openedx/core/djangoapps/plugin_api/views.py

```diff
@@ -82,6 +82,8 @@ def render_to_standalone_html(self, request, fragment, **kwargs):
         """
         Renders this fragment to HTML for a standalone page.
         """
+        if not fragment:
```

Comment:
I like the ``not`` idiom personally but I can see that there might be a situation where 0 (or some other falsey value) is passed accidentally. I'll change it to only handle None.